cmd/cmd_volume.go

```diff
@@ -225,13 +227,130 @@ func volumeTreeCommandHandler(cmd *cobra.Command, args []string) {
 	reader.Close()
 }
 
+func volumeLsCommand() *cobra.Command {
+	var cmdLs = &cobra.Command{
+		Use:   "ls <volume_name:volume_uuid> <path>",
```

Comment:
`path` should be enclosed in square brackets, since it's an optional argument

Reply:
that wouldn't work normally, because of the usage of this stuff that I would delete completely. However patched just enough to make it work for what's needed. https://github.com/nanovms/ops/blob/4ea40fb7dac1b01606a8032a81699ffee3420bf6/cmd/cmd_root.go#L66-L111